### PR TITLE
[kube-prometheus-stack] update and document grafana default admin creds

### LINKED ISSUE #506: [kube-prometheus-stack] generate a random grafana admin password by default
**Is your feature request related to a problem? Please describe.**

By default, all grafana instances generated by the kube-prometheus-stack chart will use `prom-operator` as the grafana admin password. If users expose the grafana UI publically, the interface can easily be entered, exploited, hacked etc.

**Describe the solution you'd like**

It should generate a random password by default. The grafana helm chart already supports this, we just have to update the default password to be an empty string in the values.yml

**Describe alternatives you've considered**

**Additional context**

I have proposed the changes in #505

### PR DESCRIPTION
**Pull request by adamkgray**
notes:
- explicitly expose the grafana adminUser value
- generate a psuedo-random grafana admin password by default

Signed-off-by: adamkgray <adamkgray@pm.me>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #506 

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/values.yaml [hunk 1]
```diff
@@ -580,7 +580,15 @@ grafana:
...
-  adminPassword: prom-operator
+  ## Set the default admin username
+  #
+  adminUser: admin
+
+  ## Set the default admin password
+  ## An empty string generates a psuedo-random password
+  ## Otherwise, this value is used
+  #
+  adminPassword: ""
```
Comment (monotek) [inline charts/kube-prometheus-stack/values.yaml:591]: imho thats a breaking change, even if nobody should use a static password here.
not sure if it should be a bugfix or major breaking change?

please also raise chart version.

**Review by monotek — CHANGES_REQUESTED** [review]
.

**Comment by scottrigby** [comment]
Please see this comment https://github.com/prometheus-community/helm-charts/issues/506#issuecomment-748295501

**Review by monotek — CHANGES_REQUESTED** [review]
Please rebase master / update version.

**Comment by vsliouniaev** [comment]
I don't think it's a good idea to change the defaults like this. They are currently inheriting defaults from the upstream grafana chart, so if it doesn't make sense to do there I don't think it makes sense to do here

**Pull request closed without merging**